src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Lottos {
+    private static final int INITIAL_COUNT = 0;
+
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public Map<MatchResult, Integer> createMatchResults(WinningNumbers winningNumbers, BonusNumber bonusNumber) {
```

Comment:
```suggestion
    public LottoResults createMatchResults(WinningNumbers winningNumbers, BonusNumber bonusNumber) {
```
로또 결과를 반환하게 하면 LottoResults 클래스 외에 다른 코드에서 Map<MatchResult, Integer>를 복잡하게 다룰 일이 줄어들거에요
LottoResults에서만 Map<MatchResult, Integer>를 다루도록 코드를 변경해보아요